packages/extensions-sdk/src/cli/commands/validators/check-built-code.ts

```diff
@@ -0,0 +1,53 @@
+import { EXTENSION_PKG_KEY } from '@directus/extensions';
+import fse from 'fs-extra';
+import { type Ora } from 'ora';
+import type { Report } from '../../types.js';
+
+const checkBuiltCode = {
+	name: 'built-code',
+	handler: async (spinner: Ora, reports: Array<Report>) => {
+		spinner.text = 'Check for built code';
```

Comment:
How does the spinner handle multiple, concurrent updates of the `text` that are happening because the handlers all run at the same time? will is just flash whatever handler is run last? If so maybe we can make all the handlers sync and just do them in sequence?

`fs-extra` has sync variants of all the call's we are making here and it's definitely not a performance issue for checking a couple of files.

Reply:
If we switched to sync it would probably still flash through them and report the success/failure as it does now?